src/Listeners/Database/MigratesTenants.php

```diff
@@ -47,14 +47,23 @@ public function subscribe(Dispatcher $events)
      */
     public function migrate(WebsiteEvent $event): bool
     {
-        $path = config('tenancy.db.tenant-migrations-path');
+        $paths = $this->getMigrationPaths();
 
-        if ($path && realpath($path) && $this->connection->migrate($event->website, $path)) {
-            $this->emitEvent(new Events\Websites\Migrated($event->website));
-
-            return true;
+        foreach ($paths as $path) {
+            if ($path && realpath($path) && $this->connection->migrate($event->website, $path)) {
+                $this->emitEvent(new Events\Websites\Migrated($event->website));
+            }
         }
 
         return true;
     }
+
+    protected function getMigrationPaths()
+    {
+        if (($path = config('tenancy.db.tenant-migrations-path')) && ! empty($path)) {
+            return (array) $path;
```

Comment:
Wouldn't this make it an array of arrays of paths? That doesn't seem to be what you're looking for.

Reply:
im not understand what u mean by that..

Reply:
@fletch3555 this would only apply in case the configuration would be an array of empty strings or something; I think that would be very unlikely..

Reply:
Apologies, you're correct.  I was thinking it was `return [$path]`, which would wrap whatever `$path` is in an array, which would make an array of arrays if that config value is already an array.

Instead, this is casting it as an array, which should be fine.